Lib/tcl/tclwstrings.swg

```diff
@@ -44,11 +44,11 @@ SWIGINTERNINLINE Tcl_Obj *
 SWIG_FromWCharPtrAndSize(const wchar_t* carray, size_t size)
 {
   Tcl_Obj *res = NULL;
-  if (size < INT_MAX) {
+  if (size < TCL_SIZE_MAX) {
     Tcl_Encoding encoding = NULL;
     char *src = (char *) carray;
-    int srcLen = (int)(size*sizeof(wchar_t));
-    int dstLen = (int)(size*sizeof(Tcl_UniChar));
+    Tcl_Size srcLen = (Tcl_Size)(size*sizeof(wchar_t));
+    Tcl_Size dstLen = (Tcl_Size)(size*sizeof(Tcl_UniChar));
```

Comment:
This is an pre-existing problem and something to deal with separately, but I notice these multiplications can overflow.  Perhaps the check above should be:

```
if (size < TCL_SIZE_MAX / sizeof(wchar_t) && size < TCL_SIZE_MAX / sizeof(Tcl_UniChar)) {
```